src/ResourceManager/KeyVault/Commands.KeyVault/Models/PSKeyVaultKeyAttributes.cs

```diff
@@ -26,8 +26,7 @@ public class PSKeyVaultKeyAttributes
         public PSKeyVaultKeyAttributes()
         { }
 
-        internal PSKeyVaultKeyAttributes(bool? enabled, DateTime? expires, DateTime? notBefore, string keyType,
-            string[] keyOps, Hashtable tags)
+        internal PSKeyVaultKeyAttributes(bool? enabled, DateTime? expires, DateTime? notBefore, string keyType, string[] keyOps, Hashtable tags)
```

Comment:
Did you mean to change this include the size?

Reply:
No, this is intentional. The size remains strictly a creation-time parameter, and is not captured in any client-side data type. This matches the behavior of the SDK, REST API and the service, respectively.